<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fourier Transform</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,40,40">
    <item row="1" column="0">
     <widget class="QCustomPlot" name="sourceGraph" native="true"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,50,0,7,0,20">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>y =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="equationText">
         <property name="text">
          <string>sin(2*x)+cos(7*x)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>N = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sampleSizeText">
         <property name="text">
          <string>64</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Discrete</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fast</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>In Frequency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Time</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCustomPlot" name="phaseGraph" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QCustomPlot" name="resultGraph" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="QCustomPlot" name="amplitudeGraph" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
